static/styles/stats.css

```diff
@@ -279,6 +279,17 @@ svg {
     margin: 0px 0px 8px 0px;
 }
 
+#full_update_anchor {
+    display: none;
+    color: lightslategray;
```

Comment:
We also have other grays in the product it might make sense to crib from, like `#aaa` (the color of "press enter to send"). I like the current look though, so if that doesn't look good we should find another existing color that's closer to lightslategray.